.eslintrc

```diff
@@ -15,11 +20,25 @@
     "node": true
   },
   "rules": {
+    "prettier/prettier": [
```

Comment:
We need to move this back into the `.prettierrc` file. If you have an empty `rc` file it will use default settings. Then this tells it to error if this setting are not met and these setting conflict with prettiers default settings. 

Try to save a js file within this repo and you will see what I am talking about

Reply:
I found there were conflicts due to my incorrect `fix:code` script. It is now changed, and is ran when we run `yarn run develop`. I've tested it on my end, seems to be the fix.

Reply:
That did not work for me